Formula/linux-pam.rb

```diff
@@ -4,27 +4,26 @@ class LinuxPam < Formula
   url "https://github.com/linux-pam/linux-pam/releases/download/v1.5.2/Linux-PAM-1.5.2.tar.xz"
   sha256 "e4ec7131a91da44512574268f493c6d8ca105c87091691b8e9b56ca685d4f94d"
   license any_of: ["BSD-3-Clause", "GPL-1.0-only"]
+  revision 1
   head "https://github.com/linux-pam/linux-pam.git", branch: "master"
 
   bottle do
     sha256 x86_64_linux: "5beee64dc646a1b870bee7b07259cbf2eb66146d2856233f5a17c69db709be4d"
   end
 
   depends_on "pkg-config" => :build
-  depends_on "berkeley-db"
```

Comment:
Need someone more familiar with licensing to comment, but there appears to be some restrictions/issues with using `AGPL-3.0-only` `berkeley-db` as dependency.

Due to this, most Linux distros have not updated `db`/`libdb`/`berkeley-db` beyond 5.x, e.g.
- Arch: https://archlinux.org/packages/core/x86_64/db/
- Debian: https://packages.debian.org/sid/libdb-dev
- Fedora: https://src.fedoraproject.org/rpms/libdb

For `linux-pam`, distros either use the old version or drop the dependency. Following ArchLinux here in dropping dependency (See https://github.com/archlinux/svntogit-packages/blob/packages/pam/trunk/PKGBUILD#L34)

Reply:
I agree to not use `berkeley-db` unless necessary. macOS ships with db 1.8, which should be used unless a newer one is absolutely necessary.

We do have `berkeley-db@4` too, though I'd still side on disabling db support entirely.